### ISSUE TITLE
Release of full evaluation artifact

### ISSUE THREAD

**Issue by kshxtij** [VicenteVivan/geo-clip#18]
Would it be possible for the full training code including the loss functions to be made public, alongside some of the evaluation code used to generate the plots in section 7.1 and 7.4.

**Comment by Larry-Gan** [comment]
The training code works out of the box, and gets decent results. You'll need to add your own pipeline to pre-compute embeddings though. Also Siglip has better performance than Clip
